packages/next/src/server/request/draft-mode.ts

```diff
@@ -311,7 +184,7 @@ function trackDynamicDraftMode(expression: string, constructorOpt: Function) {
 
     if (workStore.dynamicShouldError) {
       throw new StaticGenBailoutError(
-        `Route ${workStore.route} with \`dynamic = "error"\` couldn't be rendered statically because it used \`${expression}\`. See more info here: https://nextjs.org/docs/app/building-your-application/rendering/static-and-dynamic#dynamic-rendering`
+        `Route ${workStore.route} with \`dynamic = "error"\` couldn't be rendered statically because it used \`${expression}\`. \`draftMode()\` returns a Promise and must be unwrapped with \`await\` or \`React.use()\` before accessing its properties. See more info here: https://nextjs.org/docs/app/building-your-application/rendering/static-and-dynamic#dynamic-rendering`
```

Comment:
I missed this in the earlier reviews but the original error was correct. this error happens after you have awaited draftMode and then have called `.enable()` or `.disable()`. so you won't see it in the context of sync accessing properties on the promise.